assets/sass/helpers/_new_tab_links.scss

```diff
@@ -0,0 +1,27 @@
+// The following classes are used in assets/js/aplus.js. Therefore, you must
+// change the name of the classes in both files.
+
+.icon::after {
+	display: inline-block;
+	font-style: normal;
+	font-variant: normal;
+	text-rendering: auto;
+	-webkit-font-smoothing: antialiased;
+}
+
+.external-link::after {
+	font-family: "Font Awesome 5 Free";
```

Comment:
Did you download a font library (Font Awesome) for this to work? I don't see any library in the pull request.

Reply:
There is a link in the **base.html** file https://github.com/apluslms/a-plus/blob/6a3d1e5342f234fc85f915e66a31b099093cc609/templates/base.html#L21